src/main/java/javaClasses/homework_4/AndranikSargsyan/QualityEngineer.java

```diff
@@ -0,0 +1,17 @@
+package javaClasses.homework_4.AndranikSargsyan;
+
+public class QualityEngineer extends Staff{
+
+    private String fieldType;
+
+    public QualityEngineer(String field){
+        this.fieldType = field;
+    }
+
+    @Override
+    public String doWork() {
+        setName("Jon");
+        setCompany("EGS");
```

Comment:
Why you set name and company inside doWork? Please change, it just only need to print info, without setting it